repositories.py

```diff
@@ -43,9 +43,9 @@
     ],
     [
         "group g",
-        "<name>",
-        ["https://github.com/<gh_id>/<proj_id>", ],
-        "<minitwit_url>",
+        "DevJanitors",
+        ["https://github.com/lauritsbonde/DevJanitors", ],
+        "https://github.com/lauritsbonde/DevJanitors/blob/master/minitwit.py",
```

Comment:
This line will be a link to your live server later, so keep it with `<minitwit_url>` for now.

Reply:
Looks fine if you remove this line with the .py file